cli/src/trusted_operation.rs

```diff
@@ -135,54 +136,59 @@ fn send_request(
 		block_hash
 	);
 	info!("Waiting for execution confirmation from enclave...");
-	let mut subscription = chain_api.subscribe_system_events().unwrap();
-
+	let mut subscription = chain_api.subscribe_events().unwrap();
 	loop {
-		let ret: ProcessedParentchainBlockArgs =
-			chain_api.wait_for_event(&mut subscription).unwrap();
```

Comment:
`wait_for_event` was removed from the api client because with `next_event` available, not much functionality was won with it and the danger of a forever loop was not very user friendly.